user_management/api/serializers.py

```diff
@@ -74,11 +77,20 @@ def restore_object(self, attrs, instance=None):
 
 class PasswordChangeSerializer(serializers.ModelSerializer):
     old_password = serializers.CharField(
-        write_only=True, label=_('Old password'))
+        write_only=True,
+        label=_('Old password'),
+    )
     new_password = serializers.CharField(
-        write_only=True, min_length=8, label=_('New password'))
+        write_only=True,
+        min_length=8,
+        label=_('New password'),
+        validators=[validate_password_strength],
+    )
     new_password2 = serializers.CharField(
-        write_only=True, min_length=8, label=_('Repeat new password'))
+        write_only=True,
+        min_length=8,
```

Comment:
Does this number need to be extracted as a constant as we have it repeated in `RegistrationSerializer`. Having one constant would help changing it later.